src/coreclr/inc/readytorun.h

```diff
@@ -20,7 +20,7 @@
 // If you update this, ensure you run `git grep MINIMUM_READYTORUN_MAJOR_VERSION`
 // and handle pending work.
 #define READYTORUN_MAJOR_VERSION 10
-#define READYTORUN_MINOR_VERSION 0x0001
+#define READYTORUN_MINOR_VERSION 0x0002
```

Comment:
This should not be necessary. The helpers that you are deleting were never exposed as R2R helpers.

Reply:
Thanks, reverted